chall01/mlindhol.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+import sys
+
+
+def error():
+    print("usage: ./mlindhol.py <a-zA-Z string>")
+    exit(-1)
+
+
+if len(sys.argv) != 2 or len(sys.argv[1]) == 0:
+    error()
+
+morse_array = [".-", "-...", "-.-.", "-..", ".", "..-.", "--.", "....", "..", ".---", "-.-", ".-..", "--",
+               "-.", "---", ".--.", "--.-", ".-.", "...", "-", "..-", "...-", ".--", "-..-", "-.--", "--.."]
+
+input_str = sys.argv[1]
+
+for x in input_str:
+    av = ord(x)
+    if av < 32 or 32 < av < 65 or 90 < av < 97 or av > 122:
+        error()
```

Comment:
```python
if not x.isalpha():
  error()
```
more pythonic way to do that.

Reply:
isalpha() accepts ä, ö and å (and other scandic letters i think) but there is no equivalent morse code for those that is why I change it to this if-madness. But that is nice short way of doing it. This was my original way of doing it: 
```
if any(x.isalpha() is False and x.isspace() is False for x in input_str)
    error()
```